galaxy_ng/app/api/v3/serializers/collection.py

```diff
@@ -53,15 +53,15 @@ def get_versions_url(self, obj):
 
     def get_highest_version(self, obj):
         """Get a highest version and its link."""
-        collection = self.context["highest_versions"][obj.pk]
+        data = super().get_highest_version(obj)
         kwargs = {
             "path": self.context["path"],
-            "namespace": collection.namespace,
-            "name": collection.name,
-            "version": collection.version,
+            "namespace": obj.namespace,
+            "name": obj.name,
+            "version": data['version'],
         }
-        href = self._get_href("collection-versions-detail", **kwargs)
-        return {"href": href, "version": str(collection.version)}
+        data['href'] = self._get_href("collection-versions-detail", **kwargs)
+        return data
```

Comment:
probably doesn't matter, but does `data` dictionary now return more than just `href` and `version` keys?

Reply:
@awcrosby no, on pulp_ansible it returns only href and version

https://github.com/pulp/pulp_ansible/blob/04df5e5ad059f39c5c2c8cd37d8472040af22611/pulp_ansible/app/galaxy/v3/serializers.py#L82